OneApply/urls.py

```diff
@@ -11,6 +11,7 @@
     path("admissions/", include("admissions.urls")),
     path("dashboard/", include("dashboard.urls")),
     path("high_school/", include("high_school.urls")),
+    path("recommendation/", include("recommendation.urls")),
```

Comment:
Maybe we shouldn't include the urls for applications under dashboard globally. I'm not sure if it'll work and plus the high_school part is added, so if it's possible just check this locally and remove them if they're redundant. You can choose to ignore this and add a TODO instead if that's more convenient wrt the time frame.

Reply:
done

Reply:
I had to add it back in or else it gave errors